fiveofswords/botc-bot
# Updated the bot to use JSON rather than a pickle for global settings

For now just assumes that preferences.json lives where preferences.pckl lived.  Can update to be a preference in a followup change.

Manually tested:
- Creating an alias
- Using alias
- Setting default vote
- Changing set default vote
- Clearing default vote
- Trying to clear default vote if none set
- Default vote automaticaly votes expected value after expected time (for not traveler path, did not manually test traveler path)
